File: fontes/br.com.wdc.shopping/br.com.wdc.shopping.view.remote/remote.shell.react/src/scripts/views/login/login-sel.ts
/**
 * Seletores CSS do módulo de login.
 * Importado por LoginView.tsx — não referenciar login.scss diretamente nas views.
 */
import GlobalSel from '@root/global-sel'

import './login.scss'

const LoginSel = {
  ...GlobalSel,

  // :: Layout

  root: 'wdc-login',
  leftPanel: 'wdc-login__left-panel',
  contentCenter: 'wdc-login__content-center',
  logoIconLg: 'wdc-login__logo-icon-lg',
  titleLg: 'wdc-login__title-lg',
  subtitleLg: 'wdc-login__subtitle-lg',
  featuresList: 'wdc-login__features-list',
  featureRow: 'wdc-login__feature-row',
  featureIcon: 'wdc-login__feature-icon',
  featureText: 'wdc-login__feature-text',

  // :: Form panel

  formPanel: 'wdc-login__form-panel',
  formContent: 'wdc-login__form-content',

  // :: Mobile header

  mobileLogo: 'wdc-login__mobile-logo',
  mobileCircle1: 'wdc-login__mobile-circle-1',
  mobileCircle2: 'wdc-login__mobile-circle-2',
  mobileContent: 'wdc-login__mobile-content',
  logoBoxSm: 'wdc-login__logo-box-sm',
  iconSm: 'wdc-login__icon-sm',
  mobileTitle: 'wdc-login__mobile-title',
  mobileSubtitle: 'wdc-login__mobile-subtitle',

  // :: Welcome + fields

  welcomeWrap: 'wdc-login__welcome-wrap',
  welcomeTitle: 'wdc-login__welcome-title',
  welcomeSubtitle: 'wdc-login__welcome-subtitle',
  fieldLabel: 'wdc-login__field-label',
  field: 'wdc-login__field',
  fieldPassword: 'wdc-login__field-password',
  enterBtn: 'wdc-login__enter-btn',

  // :: Demo hint

  demoHint: 'wdc-login__demo-hint',
  demoText: 'wdc-login__demo-text',
  demoHighlight: 'wdc-login__demo-highlight',
} as const

export default LoginSel

File: fontes/br.com.wdc.shopping/br.com.wdc.shopping.view.remote/remote.shell.codenameone/common/src/main/java/br/com/wdc/shopping/view/remote/shell/cn1/views/login/LoginSel.java
package br.com.wdc.shopping.view.remote.shell.cn1.views.login;

import br.com.wdc.shopping.view.remote.shell.cn1.Sel;

/** Seletores (UIIDs) da tela de login — espelha {@code views/login/_login.scss}. Herda os globais. */
public final class LoginSel extends Sel {

    public static final LoginSel INSTANCE = new LoginSel();

    private LoginSel() {
        // singleton
    }

    public final String LOGIN_CARD = "LoginCard";
    public final String LOGIN_HERO = "LoginHero";
    public final String LOGIN_BANNER = "LoginBanner";
    public final String LOGO_BOX = "LogoBox";
    public final String BANNER_TITLE = "BannerTitle";
    public final String BANNER_SUBTITLE = "BannerSubtitle";
    public final String FEATURE_TEXT = "FeatureText";
    public final String WELCOME_TITLE = "WelcomeTitle";
    public final String WELCOME_SUBTITLE = "WelcomeSubtitle";
    public final String FIELD_LABEL = "FieldLabel";
    public final String DEMO_HINT = "DemoHint";
    public final String DEMO_TEXT = "DemoText";
}
